### PR TITLE
feat(store-engine,store-engine-bro-fs): Distribute FileSystemEngine in separate package

### PR DESCRIPTION
**Pull request by bennyn**

### CODE REVIEW

File: packages/store-engine-bro-fs/package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    "jasmine": "3.4.0",
+    "karma": "4.1.0",
+    "karma-chrome-launcher": "2.2.0",
+    "karma-jasmine": "2.0.1",
+    "karma-typescript": "4.1.0",
+    "rimraf": "2.6.3",
+    "ts-node": "8.3.0",
+    "typescript": "3.5.2"
+  },
+  "description": "",
```
Comment (ffflorian) [inline packages/store-engine-bro-fs/package.json]: Let's add a description

File: packages/store-engine-bro-fs/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+  private config: FileSystemEngineOptions = {
+    fileExtension: '.dat',
+    size: TEN_MEGABYTES,
+    type: typeof window === 'undefined' ? 0 : window.TEMPORARY,
+  };
+
+  constructor() {}
+
+  public async isSupported(): Promise<void> {
```
Comment (ffflorian) [inline packages/store-engine-bro-fs/src/index.ts]:
```suggestion
  async isSupported(): Promise<void> {
```

File: packages/store-engine-bro-fs/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+    if (typeof window === 'undefined' || !fs.isSupported()) {
+      const message = `File and Directory Entries API is not available on your platform.`;
```
Comment (ffflorian) [inline packages/store-engine-bro-fs/src/index.ts]: "File and Directory Entries API"?

File: packages/store-engine-bro-fs/src/index.ts [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+      throw new UnsupportedError(message);
+    }
+  }
+
+  public async init(storeName = '', options?: FileSystemEngineOptions): Promise<FileSystem> {
```
Comment (ffflorian) [inline packages/store-engine-bro-fs/src/index.ts]:
```suggestion
  async init(storeName = '', options?: FileSystemEngineOptions): Promise<FileSystem> {
```

File: packages/store-engine-bro-fs/src/index.ts (hunk 3 above)
Comment (bennyn) [reply packages/store-engine-bro-fs/src/index.ts]: That's what it is called: https://developer.mozilla.org/en-US/docs/Web/API/File_and_Directory_Entries_API

Comment (ffflorian) [reply packages/store-engine-bro-fs/src/index.ts]: Ah okay :)

File: packages/store-engine-bro-fs/package.json (hunk 1 above)
Comment (bennyn) [reply packages/store-engine-bro-fs/package.json]: Done!

**Pull request merged**